dataikuapi/dss/modelevaluationstore.py

```diff
@@ -110,12 +110,13 @@ def delete(self):
 
 
     ########################################################
-    # Runs
+    # Model evaluations
     ########################################################
 
     def list_model_evaluations(self, as_type=None):
         """
-        List the model evaluations in this model evaluation store.
+        List the model evaluations in this model evaluation store. The list is sorted
```

Comment:
Same comment as above for `project.list_model_evaluation_stores()`: we should return a list of `DSSModelEvaluation`, this makes the API much easier to use & discover.

Reply:
Ditto.